extensions/authentication-passport/README.md

```diff
@@ -0,0 +1,245 @@
+# Passport Strategy Adapter
```

Comment:
I wonder if we should mark it as experimental? See an example at https://github.com/strongloop/loopback-next/blob/prometheus/extensions/metrics/README.md#stability-warningexperimentalwarning.

Reply:
👍 Added.